components/ActivityIndicator/docs/README.md

```diff
@@ -1,5 +1,7 @@
 # Circular progress & activity indicator
 
+<!-- badges -->
```

Comment:
What does this do?

Reply:
This informs the readme generator that it should generate the badges for this component here.